dungeon/chain.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
you'll also need chain_type (cosmos)
and that may require the bech32 prefix property as well